modules/preview-react/pill/lib/Pill.tsx

```diff
@@ -147,8 +138,15 @@ const StyledBasePill = styled(BaseButton.as('button'))<StyledType & PillProps>(
     }),
   },
   ({variant}) => ({
-    '&:focus': {
+    '&:focus, &:focus-visible': {
```

Comment:
does this need the `.focus` class?

Reply:
Yeah this just got updated. I made `focus` a class and `focus-visible` the psuedo